pasta_eln/database.py

```diff
@@ -567,7 +567,9 @@ def saveView(self, designName:str, viewCode:dict[str,str]) -> None:
     try:
       designDoc.save()
     except Exception:
-      print('**ERROR dsv01: something unexpected has happend. Log-file has traceback')
+      print('**ERROR dsv01: something unexpected has happend.')
```

Comment:
Type in the error message, "happened"..please correct and also so we only log the exception right?

Reply:
I corrected the spelling. The exception is written into the log for easy sending to developers, inspection,... The terminal gives just a error message to make the user more aware. Yes, a pop-up on the screen would be even better. Future issue.